release-notes/3.0/preview/api-diff/preview5/3.0-preview5_System.Runtime.Intrinsics.X86.md

```diff
@@ -0,0 +1,293 @@
++        public unsafe static Vector256<int> ConvertToVector256Int32(ushort* address);
++        public unsafe static Vector256<long> ConvertToVector256Int64(byte* address);
++        public unsafe static Vector256<long> ConvertToVector256Int64(short* address);
++        public unsafe static Vector256<long> ConvertToVector256Int64(int* address);
++        public static Vector256<long> ConvertToVector256Int64(Vector128<byte> value);
++        public static Vector256<long> ConvertToVector256Int64(Vector128<ushort> value);
++        public static Vector256<long> ConvertToVector256Int64(Vector128<uint> value);
++        public unsafe static Vector256<long> ConvertToVector256Int64(sbyte* address);
++        public unsafe static Vector256<long> ConvertToVector256Int64(ushort* address);
++        public unsafe static Vector256<long> ConvertToVector256Int64(uint* address);
+-        public static Vector256<ushort> ConvertToVector256UInt16(Vector128<byte> value);
+
+-        public static Vector256<uint> ConvertToVector256UInt32(Vector128<byte> value);
+
+-        public static Vector256<uint> ConvertToVector256UInt32(Vector128<ushort> value);
+
+-        public static Vector256<ulong> ConvertToVector256UInt64(Vector128<byte> value);
+
+-        public static Vector256<ulong> ConvertToVector256UInt64(Vector128<ushort> value);
+
+-        public static Vector256<ulong> ConvertToVector256UInt64(Vector128<uint> value);
+
+     }
+     public enum FloatComparisonMode : byte {
+-        EqualOrderedNonSignaling = (byte)0,
+
+-        EqualOrderedSignaling = (byte)16,
+
+-        EqualUnorderedNonSignaling = (byte)8,
+
+-        EqualUnorderedSignaling = (byte)24,
+
+-        FalseOrderedNonSignaling = (byte)11,
+
+-        FalseOrderedSignaling = (byte)27,
+
+-        GreaterThanOrderedNonSignaling = (byte)30,
+
+-        GreaterThanOrderedSignaling = (byte)14,
+
+-        GreaterThanOrEqualOrderedNonSignaling = (byte)29,
+
+-        GreaterThanOrEqualOrderedSignaling = (byte)13,
+
+-        LessThanOrderedNonSignaling = (byte)17,
+
+-        LessThanOrderedSignaling = (byte)1,
+
+-        LessThanOrEqualOrderedNonSignaling = (byte)18,
+
+-        LessThanOrEqualOrderedSignaling = (byte)2,
+
+-        NotEqualOrderedNonSignaling = (byte)12,
+
+-        NotEqualOrderedSignaling = (byte)28,
+
+-        NotEqualUnorderedNonSignaling = (byte)4,
+
+-        NotEqualUnorderedSignaling = (byte)20,
+
+-        NotGreaterThanOrEqualUnorderedNonSignaling = (byte)25,
+
+-        NotGreaterThanOrEqualUnorderedSignaling = (byte)9,
+
+-        NotGreaterThanUnorderedNonSignaling = (byte)26,
+
+-        NotGreaterThanUnorderedSignaling = (byte)10,
+
+-        NotLessThanOrEqualUnorderedNonSignaling = (byte)22,
+
+-        NotLessThanOrEqualUnorderedSignaling = (byte)6,
+
+-        NotLessThanUnorderedNonSignaling = (byte)21,
+
+-        NotLessThanUnorderedSignaling = (byte)5,
+
++        OrderedEqualNonSignaling = (byte)0,
```

Comment:
For reference, these were renamed, as were the `Sse.Compare*` methods, to be more consistent and to try and make the names less confusing/match the underlying semantics better.